### DISCUSSION TITLE
Model xxts_v2 always downloads (solution)

### DISCUSSION THREAD

**Discussion by marioguima** [coqui-ai/TTS#3304] [General]
Is anyone also experiencing the same behavior when using the tts_models/multilingual/multi-dataset/xtts_v2 model that always downloads?

**Comment by Tessory** [comment]
Also had that issue. I installed TTS with XTTS_v2 model on three computer. And on one of them have same issue. Only found way to turn off internet when run xttsv2. Hope somebody found better solution or will be fix.

**Comment by Clad3815** [comment]
Maybe because you have not the lastest TTS package installed so the hash is not correct

**Comment by marioguima** [comment]
Exactly. Now that I see that the version has updated.
I was debugging the code and saw that the hash test failed.
The old one was this 5ce0502bfe3bc88dc8d9312b12a7558c the new one was this 10f92b55c512af7a8d39d650547a15a7
That was exactly it.
In this case, we just need to update with git pull or something similar if the person downloaded directly using zip

**Comment by erew123** [comment]
Please see here https://github.com/coqui-ai/TTS/discussions/3301#discussioncomment-7663904 includes reverting the model back if you dont like the way the new model sounds.

**Comment by marioguima** [comment]
Thanks for everything @erew123
In my case the problem was just the need to update
I didn't even compare the sound to how it was before.
Again, thank you very much

**Comment by erogol** [comment]
`pip install -U TTS `

**Comment by marioguima** [comment]
In my case, I need to download the source code again as I am working on the clone in my branch.
Thank you very much for your help, this command
`pip install -U TTS`
will definitely help people

**Comment by marioguima** [comment]
Just to register.
The solution in my case:
Observation. I forked this repository
To update your fork, follow these steps
1) Commit your changes to your branch, in your forked repository
`git commit -m "put your comment"`
2) Add the original project with
`git remote add upstream https://github.com/coqui-ai/TTS.git`
3) Return to your branch
`git checkout main`
4) Sync projects
`git pull dev upstream`
